NST/src/main/java/com/ebay/service/logger/platforms/util/PlatformTestFileManager.java

```diff
@@ -52,6 +51,15 @@ public PlatformTestFileManager() throws FileNotFoundException {
       for (File file : files) {
         filesInPlatformTestDirectory.add(file);
       }
+      break;
+    case ANDROID_KOTLIN:
```

Comment:
The implementation of this switch case is identical to the Android one. Please collapse the two so they drop through into the same case implementation and we don't have duplicate code.

Reply:
I could have but I created a new KotlinFileFilter which is distinctly taking .kt files in it from Android_Kotlin case, I could modify the method to take which Android it is but I feel keeping it separate makes sense since we will have 2 Android variants one for Java vs Kotlin. Let me know if that is okay

Reply:
Done.